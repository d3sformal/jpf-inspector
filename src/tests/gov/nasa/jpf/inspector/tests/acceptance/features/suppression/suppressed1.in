disable thread 1

create breakpoint property_violated

run

finish
