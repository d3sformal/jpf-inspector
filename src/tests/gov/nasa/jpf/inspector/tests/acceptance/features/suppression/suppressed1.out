The thread 1 is now disabled.
INFO: JPF created and connected, SuT is started
INFO: JPF Terminating